content/criteria.csv

```diff
@@ -4,19 +4,19 @@ deceased_died_of_COVID,The deceased died or likely died of COVID-19.,boolean,TRU
 deceased_death_location_is_US,"The deceased died in the U.S., including its territories.",boolean,TRUE,
 deceased_public_safety_officer,The deceased was a public safety officer who died in the line of duty.,boolean,TRUE,
 deceased_date_of_death,The deceased died on the following date,boolean,TRUE,
-deceased_served_in_active_military,"The deceased served in the active military, naval, or air service and",radio,was discharged under conditions other than dishonorable; died while on active duty; had retired from the service; not applicable,
+deceased_served_in_active_military,"The deceased served in the active military, naval, or air service and",radio,not applicable; was discharged under conditions other than dishonorable; died while on active duty; had retired from the service,

```

Comment:
@lgarceau768 I think we want the `not applicable` option to be the last option for radio options.

Reply:
Fixed!